sorc/ncep_post.fd/SURFCE.f

```diff
@@ -937,7 +940,7 @@ SUBROUTINE SURFCE
       ENDIF
 !
 !     ACM GRID SCALE SNOW AND ICE
-      IF ( IGET(244)>0 ) THEN
+      IF ( IGET(244)>0 .and. .false. ) THEN
```

Comment:
@jaymes-kenyon @Jeff-Duda @BenjaminBlake-NOAA  The UPP reads accumulated grid scale snow/ice ('SNOWNC') from MPAS output but has not yet assigned a GRIB2 name to output it as GRIB2 yet. I suggest changing UPP ID to an unused number, such as 248 and addressing the output of accumulated grid scale snow/ice in GRIB2 in the future. Please let me know your thoughts.
```suggestion
      IF ( IGET(248>0 ) THEN
```